DeviceTests/DeviceTests.swift

```diff
@@ -15,99 +15,103 @@ private extension DeviceType {
+
+        case .iPhone7: ["iPhone9,1", "iPhone9,3"]
+        case .iPhone7Plus: ["iPhone9,2", "iPhone9,4"]
+
+        case .iPhone8: ["iPhone10,1", "iPhone10,4"]
+        case .iPhone8Plus: ["iPhone10,2", "iPhone10,5"]
+        case .iPhoneX: ["iPhone10,3", "iPhone10,6"]
+
+        case .iPhoneXS: ["iPhone11,2"]
+        case .iPhoneXSMax: ["iPhone11,4", "iPhone11,6"]
+        case .iPhoneXR: ["iPhone11,8"]
+
+        case .iPhone11: ["iPhone12,1"]
+        case .iPhone11Pro: ["iPhone12,3"]
+        case .iPhone11ProMax: ["iPhone12,5"]
+
+        case .iPhoneSEGen2: ["iPhone12,8"]
+
+        case .iPhone12Mini: ["iPhone13,1"]
+        case .iPhone12: ["iPhone13,2"]
+        case .iPhone12Pro: ["iPhone13,3"]
+        case .iPhone12ProMax: ["iPhone13,4"]
+
+        case .iPodTouch1G: ["iPod1,1"]
+        case .iPodTouch2G: ["iPod2,1"]
+        case .iPodTouch3G: ["iPod3,1"]
+        case .iPodTouch4G: ["iPod4,1"]
+        case .iPodTouch5G: ["iPod5,1"]
+        case .iPodTouch6G: ["iPod7,1"]
+        case .iPodTouch7G: ["iPod9,1"]
+
+        case .iPad: ["iPad1,1", "iPad1,2"]
+        case .iPad2: ["iPad2,1", "iPad2,2", "iPad2,3", "iPad2,4"]
+        case .iPad3: ["iPad3,1", "iPad3,2", "iPad3,3"]
+        case .iPad4: ["iPad3,4", "iPad3,5", "iPad3,6"]
+        case .iPad5: ["iPad6,11", "iPad6,12"]
+        case .iPad6: ["iPad7,5", "iPad7,6"]
+        case .iPad7: ["iPad7,11", "iPad7,12"]
+        case .iPad8: ["iPad11,6", "iPad11,7"]
+        case .iPadMini: ["iPad2,5", "iPad2,6", "iPad2,7"]
+        case .iPadMiniRetina: ["iPad4,4", "iPad4,5", "iPad4,6"]
+        case .iPadMini3: ["iPad4,7", "iPad4,8", "iPad4,9"]
+        case .iPadMini4: ["iPad5,1", "iPad5,2"]
+        case .iPadMini5: ["iPad11,1", "iPad11,2"]
+        case .iPadAir: ["iPad4,1", "iPad4,2", "iPad4,3"]
+        case .iPadAir2: ["iPad5,3", "iPad5,4"]
+        case .iPadAir3: ["iPad11,3", "iPad11,4"]
+        case .iPadAir4: ["iPad13,1", "iPad13,2"]
+        case .iPadPro9p7Inch: ["iPad6,3", "iPad6,4"]
+        case .iPadPro10p5Inch: ["iPad7,3", "iPad7,4"]
+        case .iPadPro11Inch: ["iPad8,1", "iPad8,2", "iPad8,3", "iPad8,4"]
+        case .iPadPro11Inch2: ["iPad8,9", "iPad8,10"]
+        case .iPadPro12p9Inch: ["iPad6,7", "iPad6,8"]
+        case .iPadPro12p9Inch2: ["iPad7,1", "iPad7,2"]
+        case .iPadPro12p9Inch3: ["iPad8,5", "iPad8,6", "iPad8,7", "iPad8,8"]
+        case .iPadPro12p9Inch4: ["iPad8,11", "iPad8,12"]
+        case .iPhone13Mini: ["iPhone14,4"]
+        case .iPhone13: ["iPhone14,5"]
+        case .iPhone13Pro: ["iPhone14,2"]
+        case .iPhone13ProMax: ["iPhone14,3"]
+        case .iPhoneSEGen3: ["iPhone14,6"]
+        case .iPhone14: ["iPhone14,7"]
+        case .iPhone14Plus: ["iPhone14,8"]
+        case .iPhone14Pro: ["iPhone15,2"]
+        case .iPhone14ProMax: ["iPhone15,3"]
+        case .iPhone15: ["iPhone15,4"]
+        case .iPhone15Plus: ["iPhone15,5"]
+        case .iPhone15Pro: ["iPhone16,1"]
+        case .iPhone15ProMax: ["iPhone16,2"]
```

Comment:
Can you move the iPhones above next to the other iPhones, for easier readability of this list?

Reply:
Yep, and will do the same for the other iPads I just added

Reply:
Done and added some missing iPad devices as well